start/sw/web-app/about.md

```diff
@@ -1,10 +1,31 @@
 ---
-label: need-help
-
 title: Web Application
 subsection: web-app
 section: start-sw
-description:
+description: Short introduction to web application development
 ---
 
-# Web Application
+# Web Application Development
+
+Web application development is a broad topic, which covers range of different technologies. Under web application, we can imagine anything from classic server-side apps that render dynamic HTML pages, to real-time audio/video apps which allow us to watch movies or videochat online. Even if we only focus on the applications that users display in the web browser, a full-stack web developer needs to know dozens of technologies. What is common for all of them though, Fedora is here to offer all needed components and great development environment.
```

Comment:
Wow, this is rerally long block of text. What I'd idealy want are 2-3 sentences max, otherwise nobody will read anything apart ' Web application development is a broad topic'. For real.

As this web is ment for 'developers', which actually know what they're looking for (or at -seen the web thing yet?). IMHO it'd make sense to have just a key concept (a bare overview to get started); not to try give them full-featured description. Some level of developer's `naivity` is sometimes good to get started also.

Reply:
Most of the text was removed, hope it looks better now.